### ISSUE TITLE
R/DEPENDS fp when using bbclasses

### ISSUE THREAD

**Issue by priv-kweihmann** [priv-kweihmann/oelint-adv#230]
foo.bb

```
DEPENDS:append = " z"
inherit bar
```

bar.bbclass

```
DEPENDS:append = "a"
```

turns into `'DEPENDS' entries should be ordered alphabetically` - this is a false positive as it can't be fixed
